commands/inventory/add_agent_mongodb_exporter.go

```diff
@@ -133,6 +138,10 @@ func init() {
 		StringVar(&AddAgentMongodbExporter.AuthenticationMechanism)
 	AddAgentMongodbExporterC.Flag("push-metrics", "Enables push metrics model flow,"+
 		" it will be sent to the server by an agent").BoolVar(&AddAgentMongodbExporter.PushMetrics)
-	AddAgentMongodbExporterC.Flag("disable-collectors",
-		"Comma-separated list of collector names to exclude from exporter").StringVar(&AddAgentMongodbExporter.DisableCollectors)
+
+	AddAgentMongodbExporterC.Flag("disable-collectors", "Comma-separated list of collector names to exclude from exporter").EnumsVar(
+		&AddAgentMongodbExporter.DisableCollectors, "diagnosticdata", "replicasetstatus", "dbstats", "topmetrics")
```

Comment:
should we really mention all possible options here?

Reply:
and as far as I see in this ticket it works different way. https://github.com/alecthomas/kingpin/pull/265
so it's not comma-separated list of collector names, you should repeat flag

Reply:
Reverted.